.github/workflows/go_test.yml

```
@@ -29,6 +29,8 @@ jobs:
           go-version: '1.21.9'
```

you'll need to change the go version here

```suggestion
          go-version: '1.22'
```